### ISSUE TITLE
Inconsistent Getting Started with Rails guide

### ISSUE THREAD

**Issue by tlatsas** [rails/rails#39920]
It seems that @radar started working on an overhaul of the Getting Started guide a while back but only the first (#38268) of the PRs batch (#38328 #38329) got merged. This resulted on the [guide](https://edgeguides.rubyonrails.org/getting_started.html) being inconsistent. More specifically, at the [hello rails section](https://edgeguides.rubyonrails.org/getting_started.html#say-hello-rails) we moved away from introducing a welcome controller, but [later in the guide](https://edgeguides.rubyonrails.org/getting_started.html#setting-the-application-home-page) it is still referenced.

What do you think is the best course of action here? The rest of the work is still there (see stale PRs above), should this be reverted or re-open the rest of the PRs? I can help with proof-reading and squashing/rebasing.

**Comment by radar** [comment]
I believe all the work is at https://github.com/radar/rails/tree/gs-old if you need it.

The refreshing of the GS guide is necessary work and I'd prefer if those PRs were re-opened and reviewed rather than marked as stale.

**Comment by jonathanhefner** [comment]
I've re-opened both #38328 and #38329, but I want to mention that it is exceedingly difficult to review such large diffs.  If there is any way to break them down into smaller PRs, that would be most helpful.

**Comment by tlatsas** [comment]
There is one more PR (#38288) that I missed in my original post, adding here for reference.

I started working with this one, I rebased master on top and squashed the minor commits (https://github.com/rails/rails/compare/master...tlatsas:getting-started-guide-improvements-2) however as @jonathanhefner metnioned it is indeed a huge task to review this. I will try to break it to down to one PR per guide section and see if this helps a bit.
